templates/default/ndbmtd.service.erb

```diff
@@ -6,6 +6,7 @@ StartLimitBurst=10
 
 [Service]
 User = <%= node['ndb']['user'] %>
+Nice = -20
```

Comment:
In https://github.com/logicalclocks/ndb-chef/blob/master/recipes/install.rb#L264 we set the limit for mysql user to maximum -19 nice. Same for priority. These need to be adjusted too

Reply:
They don't work. Systemd works.
Should i remove them?

Reply:
I don't think we should change the nice level for mysqld or ndb_mgmd.